modules/ROOT/pages/_partials/task-configure-proxy.adoc

```diff
@@ -0,0 +1,72 @@
+//tag::first-steps[]
```

Comment:
Adding comments here for the partials

Under Configure your SOAP Proxy
your should be initial capped
minor - remove "will" in step 5.
minor - remove "Optionally," from step 13. The "if" text indicates that this is optional.